File: src/main/java/com/examportal/dto/StudentQueryUpdateDTO.java
package com.examportal.dto;

import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class StudentQueryUpdateDTO {
    
    @NotBlank(message = "Response must be provided")
    private String response;
    
    @NotBlank(message = "Status must be provided")
    private String status;
}

File: Dotnet3/ExamPortal/DTOs/StudentQueryUpdateDTO.cs
using System.ComponentModel.DataAnnotations;

namespace ExamPortal.DTOs
{
    public class StudentQueryUpdateDTO
    {
        [Required(ErrorMessage = "Response must be provided")]
        public string Response { get; set; } = string.Empty;
        
        [Required(ErrorMessage = "Status must be provided")]
        public string Status { get; set; } = string.Empty;
    }
}
